test/controllers/AddAdditionalMetadataControllerSpec.scala

```diff
@@ -590,6 +599,61 @@ class AddAdditionalMetadataControllerSpec extends FrontEndTestHelper {
     }
   }
 
+  "AddAdditionalMetadataController formFieldOverrides" should {
+    "override the field when the description is not empty" in {
+      val fileMetadata: Map[String, List[FileMetadata]] = Map(
+        descriptionClosed -> List(FileMetadata(descriptionClosed, "true")),
+        description -> List(FileMetadata(description, "some value"))
+      )
+      val formField: FormField = RadioButtonGroupField(descriptionClosed, "name", "description", "", false, Seq(InputNameAndValue("Yes", "Yes")), "Yes", false)
+      val actualFormField = AddAdditionalMetadataController.formFieldOverrides(formField, fileMetadata)
+
+      val expectedField = RadioButtonGroupField(
+        descriptionClosed,
+        "name",
+        "This field cannot be edited here. You can edit the description in the <strong>Descriptive metadata</strong> step.",
+        "some value",
+        false,
+        Seq(InputNameAndValue("Yes", "Yes")),
+        "Yes",
+        false
+      )
+      actualFormField should equal(expectedField)
+    }
+
+    "not override the field when the description is empty" in {
```

Comment:
Maybe rename the test to make clear which field is being overridden: `not override the 'descriptionClosed' field when ...`